ingestion/src/metadata/orm_profiler/profiler/sampler.py

```diff
@@ -37,11 +37,18 @@ def __init__(
         self,
         session: Optional[Session],
         table: DeclarativeMeta,
-        profile_sample: Optional[float] = None,
+        profile_sample: Optional[dict] = None,
```

Comment:
Same comment than in `workflow.py`. Let's pass `profile_sample` and `profile_sample_type` as attributes of the class.